internal/netif/netif.go

```diff
@@ -50,7 +57,21 @@ func (m *netifManagerDefault) EnsureIPAddress() error {
 
 	l, err := m.LinkByName(m.devName)
 	if err != nil {
-		return xerrors.Errorf("could not get loopback interface %v", err)
+		var linkNotFoundErr netlink.LinkNotFoundError
+		if !errors.As(err, &linkNotFoundErr) && !m.manageInterface {
+			return xerrors.Errorf("could not get interface %s: %v", m.devName, err)
+		}
+
+		attrs := netlink.LinkAttrs{
+			Name: m.devName,
+			MTU:  65535,
+		}
+		dummyLink := &netlink.Dummy{LinkAttrs: attrs}
```

Comment:
Please consider inlining the attributes. There is no need to have them separately.